[WIP] Add initial decision making process to community handbook (#3940)
the-turing-way/the-turing-way: book/website/community-handbook/governance.md

(ch-governance)=
# Governance and Decision Making in *The Turing Way*

> Governance is a set of formal and informal practices through which an organisation sets goals, assigns responsibilities, establishes systems, and assesses outcomes of organisational action.

Reviewer: If governance can be both formal and informal, maybe we don't need to mention it.

Unless we want to emphasise that not all governance will be written down or clear to the community. In that case, it might be better in a separate sentence.

Maybe something like,

```suggestion
Governance is a set practices through which an community sets goals, assigns responsibilities, establishes systems, and assesses outcomes of organisational action.
Not all governance is formally described.
For example, processes may be understood by community leaders after evolving organically within a small group.
```

Like the book itself, the governance of _The Turing Way_ is always a work in progress!
We are very keen to discuss and improve our governance approaches with the involvement of our community members.

> In this section we introduce our goverance guiding principles of:

Reviewer: ```suggestion
In this section we introduce our governance guiding principles:
```

1. **Empowerment** of contributors to participate in decision making processes.
2. **Transparency** of ways of working and how decisions are made.

> 3. **Inclusion** of diverse perspectives in all areas of the project.

Reviewer: I like these principles a lot :rocket:.

> We also cover how
> Decisions within *The Turing Way* community can be made at three different levels.

Reviewer: I feel this needs an extra adjective to explain what the levels are for, maybe

```suggestion
We also cover how decisions within *The Turing Way* community can be made at three different levels of responsibility.
```

> In this chapter we detail the different levels and you can jump to that information from the list below:

Reviewer: ```suggestion
You can jump to that information from the list below:
```

Repetition of In this chapter/section

<!-- Turn these into links to later pages -->

> 1. **{ref}`Community<ch-gov-community>`** level through individual contributors
> 2. **{ref}`Maintainance**<ch-gov-maintenance>`** level through Working Groups and Delivery Groups
> 3. **{ref}`Constitutional**<ch-gov-constitutional>`** level through the Steering Council

Reviewer: ```suggestion
1. {ref}`Community<ch-gov-community>` level of individual contributors
2. {ref}`Maintainance**<ch-gov-maintenance>` level of Working Groups and Delivery Groups
3. {ref}`Constitutional**<ch-gov-constitutional>` level of the Steering Council
```

The theme should style the links.

  This icon symbolises the Steering Council.
  There's a diagonal arrow between the Community and Maintenance sections with arrows on both sides.
  The Community side is labelled "Clear pathways to join working and delivery groups".
  The Maintainance side is labelled "Decisions and notes on GitHub".
  There is also a diagonal arrow between Maintenance and Constitutional sections which is labelled "Representation at leadership".
  In the bottom right half of the image is a scriberia cartoon of a mountain of engagement.
  A sign says "Building a Healthy Community" and there are different people walking up a path to the top of the mountain.
  They all look supportive and welcoming, and like they are motivating each other up the mountain. 
---
The three layer governance structure in The Turing Way, accompanied by the mountain of engagement which is described further in the {ref}`Shared Ownership Models<cl-shared-ownership>` chapter in the Guide for Collaboration.
The Mountain of Engagement is a _The Turing Way_ project illustration by Scriberia.
Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
```

> <!-- Turn these into links to later pages -->
> 
> We also introduce our goverance guiding principles of:
> 
> 1. **Empowerment** of contributors to participate in decision making processes.
> 2. **Transparency** of ways of working and how decisions are made.
> 3. **Inclusion** of diverse perspectives in all areas of the project.
> 

Reviewer: ```suggestion
```

This section is repeated.
